packages/ag-charts-community/src/chart/interaction/regionManager.ts

```diff
@@ -76,12 +78,14 @@ export class RegionManager {
         );
 
         injectStyle(focusStyles.css, focusStyles.block);
-        this.focusIndicator = getDocument()?.createElement('div');
```

Comment:
Note: I vaguely recall having an `undefined` document in one of the tools (perhaps it was the thumbnail generator, but I do not remember). Just letting you know in case something in the CI fails.

Reply:
Tested locally, I can verify that the thumbnail generate successfully.

FYI: The intent is that in all of these non-browser cases that the tooling provide an appropriate substitute for these APIs, which is why `getDocument()` exists specifically.